### ISSUE TITLE
CSS changes for headings

### ISSUE THREAD

**Issue by alboss-cascadia** [palma4k/report#5]
Neither `<h1>` nor `<h2>` have sufficient color contrast with the background color. Both need to be lighter. Headings should have a 3-to-1 contrast ratio with the background color.

h1 is #8B0000 and has a 1.24-to-1 contrast with the background #333333. 
h2 is #0000ff and has a 1.47-to-1 contrast with the background.

In your CSS, you'd have to change h1 to #FA0000 and h2 to #6B6BFF, or pick another color. You can use this tool to make sure the colors are okay:
[https://webaim.org/resources/contrastchecker/](https://webaim.org/resources/contrastchecker/)

**Comment by palma4k** [comment]
Thank you, it looks at lot better, I'll keep that in mind.
